_data/companies/riotgames.yml

```
@@ -0,0 +1,7 @@
+---
+name: RiotGames
+wfh: Required
```

```suggestion
wfh: Required for USA and EU offices
```

How about this then?